### PR TITLE
Migrate Product Notion pages to the Contributor Handbook

### LINKED ISSUE #936: Add a Continuous Discovery overview to the Contributor Handbook
Create a Continuous Discovery overview page under Code Contributor - Product Development Process. 
The page should have `weight: 1` in the Product Development Process hierarchy.

In the context of https://github.com/medic/cht-docs/issues/895

### PR DESCRIPTION
**Pull request by andrablaj**
#895 

Closes #936 

Migrate Product Notion pages under the Contributor Handbook.

### CODE REVIEW

**Comment by andrablaj** [comment]
@craig-landry, please add the documentation about quality assistance under "Product Handbook at Medic". Thanks!

**Comment by craig-landry** [comment]
@andrablaj I've created https://github.com/medic/cht-docs/pull/919 to add the quality assistance material.

File: content/en/product-handbook/_index.md
Comment (michaelkohn) [inline content/en/product-handbook/_index.md:9]:
```suggestion
The Product team works with Medic teammates, partners, and health workers to design, build, and maintain several open source tools. We work with our partners along the entire software development life-cycle to understand problems, capture requirements, design and build modular software systems, and document everything along the way.
```

File: content/en/product-handbook/testing/quality-assistance.md
Comment (michaelkohn) [inline content/en/product-handbook/testing/quality-assistance.md:44]:
```suggestion
Once the developer has finished writing the code they test it a bit more and open a PR for a code review. The reviewer should be able to review the code with an assumption that it is well-tested already and the focus of the review can be around the code and implementation choices.
```

Comment (craig-landry) [reply content/en/product-handbook/testing/quality-assistance.md:44]: Good one, thanks @michaelkohn

**Comment by andrablaj** [comment]
@craig-landry @michaelkohn @mrjones-plip @abbyad, here is the status of the Product Notion pages migration:

-  Migrated Product Development Process pages, as they were (meaning some pages are most probably outdated). Pending questions:
    - @craig-landry A Continuous Discovery overview page would be helpful to set the context of the process. It can be added to this PR, or I can create a separate issue for future iterations. Are you available to add that page to this PR?
    - There is a link to Allies Google Drive that I didn't open to the internet, it is still Medic restricted. Not sure how to proceed. @mrjones-plip, any thoughts?
    - The Code Health process is out of date. Please add suggestions with relevant process updates.
- The Notion Product Resources page seems out of date and I need support to understand what should be added to this handbook, and what should be archived:
    - Product Management Key Definitions section. What should we keep from it?
    - Product Education - are all the links relevant, should we add any others?
    - FAQs - what are the questions to be added to the handbook?

**Comment by andrablaj** [comment]
After a quick chat with @craig-landry, here are some decisions on the questions above:

> @craig-landry A Continuous Discovery overview page would be helpful to set the context of the process. It can be added to this PR, or I can create a separate issue for future iterations. Are you available to add that page to this PR?

Will be done via https://github.com/medic/cht-docs/issues/936.

> There is a link to Allies Google Drive that I didn't open to the internet, it is still Medic restricted. Not sure how to proceed. @mrjones-plip, any thoughts?

Will keep the link restricted to Medic.

> The Code Health process is out of date. Please add suggestions with relevant process updates.

To be done in the code review of this PR.


>Product Management Key Definitions section. What should we keep from it?
Product Education - are all the links relevant, should we add any others?
FAQs - what are the questions to be added to the handbook?

All this content should be archived as out of date.

>Should I migrate the Community Growth pages as they are currently? 

We will keep the Community Growth page in Notion as it needs a review by the Community team.

File: content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md:2]: I think we should just delete this entire page as we aren't going to keep track of it/update it here.

File: content/en/contribute/medic/product-development-process/cadence-of-events.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/cadence-of-events.md:3]: I'd just call this "Cadence" or something like that (the text wraps in the left side menu so it's kinda messy).

File: content/en/contribute/medic/product-development-process/code-health.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/code-health.md:6]: This isn't really how we are doing code health anymore... while Gareth is still owning the queue, there's not a dedicated person on code health each week. Focused Groups are responsible for code health in their area of concern and these are generally on OKRs as well. This means that we can do them when we want, but they gotta get done that quarter.

File: content/en/contribute/medic/product-development-process/product-trio/product-trio-activities.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/product-trio/product-trio-activities.md:6]: A lot of this is out of date too :-|

**Review by michaelkohn — COMMENTED** [review]
There are some things that are kind of out of date (like our code health process, for example). Not sure if it's better to post out of date stuff, let it block the merge until it gets updated, or just not include the page for now.

File: content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md:2]: Do you mean that this page should stay in Notion, or should we remove it completely?

File: content/en/contribute/medic/product-development-process/code-health.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/code-health.md:6]: I know, 😥 hence my comment [here](https://github.com/medic/cht-docs/pull/918#issuecomment-1381769429).
I was hoping that someone (like you 😊) from the reviewer list could suggest changes to the outdated pages with relevant updates. 
Any chance to suggest the relevant changes? 🙏

File: content/en/contribute/medic/product-development-process/product-trio/product-trio-activities.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/product-trio/product-trio-activities.md:6]: Do you think that you can suggest changes via a PR to be merged into this branch? 🙏

File: content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md
Comment (michaelkohn) [reply content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md:2]: We can either leave it in Notion or move it to a google sheet (or baserow itself) if we want to totally get rid of the notion page.  I'm actually hoping we'll close off most of these during a hackathon-esque session in Zanzi.

File: content/en/contribute/medic/product-development-process/code-health.md
Comment (michaelkohn) [reply content/en/contribute/medic/product-development-process/code-health.md:6]: Not today but I can do this section and the product trio activities one Thursday or Friday next week if nobody else gets to it before me!

Comment (craig-landry) [reply content/en/contribute/medic/product-development-process/code-health.md:6]: I have a [PR up for an update to code health process](https://github.com/medic/cht-docs/pull/939).

File: content/en/contribute/medic/_index.md
Comment (abbyad) [inline content/en/contribute/medic/_index.md:9]: Reframing for more general audience.


```suggestion
[Medic](https://medic.org) works with CHT partners and health workers to design, build, and maintain the Community Health Toolkit and its open source tools. Medic's Product team manages the entire software development life-cycle to understand problems, capture requirements, design and build modular software systems, and document everything along the way.
```

Comment (abbyad) [reply content/en/contribute/medic/_index.md:9]: The ["no we"](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#using-we) rule will help to get the tone right for the new public home of this content.

File: content/en/contribute/medic/product-development-process/_index.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/_index.md:6]: Started by fixing the typo, but then figured we could make the description more descriptive so that it shows well on the summary page.

```suggestion
  Explanation of the Continuous Discovery process used for product development at Medic
```

File: content/en/contribute/medic/product-development-process/_index.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/_index.md:11]: Good paragraph, and trying to make it a bit more public friendly:

```suggestion
To allow teammates to truly learn from and empathize with those using the CHT the Medic Product team has cross-functional subteams, called [Focused Working Groups](/contribute/medic/product-development-process/focused-groups), each aligned with different user types. Each group is focused on a specific set of users of the CHT and has an aligned roadmap. Roadmaps are made up of defined initiatives, which map to goals and strategies. There is a big emphasis on working towards outcomes (not only *outputs*). Product teammates work together, across disciplines, to understand opportunities for improvement, find solutions that will have an impact, and then build, deploy, and measure them.
```

File: content/en/contribute/medic/product-development-process/cadence-of-events.md
Comment (abbyad) [reply content/en/contribute/medic/product-development-process/cadence-of-events.md:3]: I'd suggest renaming the file to just cadence as well, but also think that `schedule` could be more appropriate.

File: content/en/contribute/medic/product-development-process/cadence-of-events.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/cadence-of-events.md:6]: We should update with a more descriptive sentence that acts like a mini preview. For instance: 
```suggestion
    Schedule of activities for Focused Working Groups
```

This suggestion should change if the page title becomes "Schedule of activities" :)

File: content/en/contribute/medic/product-development-process/focused-groups.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/focused-groups.md:6]: These groups were originally called Focused Working Groups internally. They are sometimes called Focused Groups as shorthand in some contexts, but I think we should maintain the full term in these pages because they are more like _"working groups"_ (a generally known thing that is similar to what we have), than _"focus groups"_ (also a known thing, but completely different).

```suggestion
title: "Focused Working Groups"
linkTitle: "Focused Working Groups"
weight: 1
description: >
  Cross-functional teams designed to better serve the community
```

File: content/en/contribute/medic/product-development-process/ux-research-repo/_index.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/ux-research-repo/_index.md:9]:
```suggestion
The Medic Product team uses [baserow.io](http://baserow.io) as a place to keep product research. Baserow is very similar to [airtable.com](http://airtable.com), [getgrist.com](http://getgrist.com), and Google Sheets.
```

File: content/en/contribute/medic/product-development-process/ux-research-repo/_index.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/ux-research-repo/_index.md:11]:
```suggestion
The _UX Research Repository_ is based off of concepts from WeWork’s [Polaris](https://tsharon.medium.com/democratizing-ux-670b95fbc07f) project. You can see an example of the Polaris (which uses Airtable) [here](https://www.airtable.com/universe/expShuhNMi0Oc0xpb/polaris-ux-nuggets).
```

File: content/en/contribute/medic/product-development-process/ux-research-repo/_index.md
Comment (abbyad) [inline content/en/contribute/medic/product-development-process/ux-research-repo/_index.md:17]: How should external people gain (view) access to the UX Research Repo?

```suggestion
To gain access to the UX Research Repo, please reach out in the #product-owner-chatter Slack channel.
```

File: content/en/contribute/medic/team.md
Comment (abbyad) [inline content/en/contribute/medic/team.md:18]: As per comment above, we shouldn't use the shorthand here. Also, I imagine this may have just been a placeholder, but we should add a bit of text here to avoid just repeating the title with the link.

```suggestion
### Focused Working Groups
[Focused Working Groups](/contribute/medic/product-development-process/focused-groups)
```

**Comment by abbyad** [comment]
This is coming together nicely @andrablaj!

Reposting some of the bigger items here, otherwise they'll get lost in the comment threads, especially once resolved:
- [Avoid using "we"](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#using-we)
- "Focused Working Groups", not "Focused Groups", more details [in comment](https://github.com/medic/cht-docs/pull/918#discussion_r1070146505)

Also, the top level structure doesn't feel quite right:
> image
I'll think about it some more before making a suggestion, but if someone has ideas, please post them!

File: content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md:2]: @craig-landry has reviewed the Baserow documentation via [this PR](https://github.com/medic/cht-docs/pull/941), and removed this page completely. Thanks Craig!

File: content/en/contribute/medic/product-development-process/ux-research-repo/_index.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/ux-research-repo/_index.md]: Good question. @craig-landry suggested keeping the Baserow internal to the Medic team. I added a clarification about this in the paragraph above.

Comment (craig-landry) [reply content/en/contribute/medic/product-development-process/ux-research-repo/_index.md]: As a long(er) term goal for the UX Repo, I would love to see a point where we can make it public. Currently it has sensitive information like interview recordings and names and contact information for people we've spoken with. Baserow has functionality that can help us remove or obfuscate data though, so with some effort and careful usage we can probably have a public-facing view available.

File: content/en/contribute/medic/team.md
Comment (abbyad) [inline content/en/contribute/medic/team.md:15]: The leadership section takes too much importance, and makes me think we could drop this whole page unless we use it to add much more on the rest of the team. As it is it doesn't really have enough context to be useful for understanding the rest of the sections in the handbook.

File: content/en/contribute/medic/testing/quality-assistance.md
Comment (abbyad) [inline content/en/contribute/medic/testing/quality-assistance.md:7]: It's not clear to me why this page is in a separate section... can we move it to within _Product Development Process_?

**Comment by mrjones-plip** [comment]
> There is a link to Allies Google Drive that I didn't open to the internet, it is still Medic restricted. Not sure how to proceed. @mrjones-plip, any thoughts?

I think this is fine.  It is useful for Medic teammates and the general idea is useful for the public.  If we get requests for access from the public and we can circle back and add a public, read only example for folks to view. We could add `(private link)` if we wanted, which I do when commenting on a GH issue I know the public will view:

```markdown
this would result in a new “Interoperability” folder within the [Allies Team folder](https://drive.google.com/drive/folders/1nDem-vQ4esCoGOlUvIX0HAr45fkuURW2) (private link).
```

File: content/en/contribute/medic/product-development-process/ux-research-repo/_index.md
Comment (mrjones-plip) [inline content/en/contribute/medic/product-development-process/ux-research-repo/_index.md:26]: These are all accessible to the public with needing a Baserow login.  When it was gated in Notion, this was fine because they were public, but un-guessable URLs. With direct links, the public can either think they're invited to add thoughtful submissions (best case) or spam us with their commercial offerings (worst case).

@michaelkohn  - thoughts on this? you've done a lot more digging into baserow's features.

If we can't think of any options, I think we should remove all the "Add a new..." links and instead post a link to [Notion's quick links page](https://www.notion.so/medicmobile/UX-Research-Repository-78c4153f21f04f97bea2025d99c09836) (or a simplified version of same)

File: content/en/contribute/medic/testing/quality-assistance.md
Comment (andrablaj) [reply content/en/contribute/medic/testing/quality-assistance.md:7]: Yes, it makes sense to move it. It was created before the product development migration from Notion, hence the confusion.

File: content/en/contribute/medic/product-development-process/ux-research-repo/_index.md
Comment (michaelkohn) [reply content/en/contribute/medic/product-development-process/ux-research-repo/_index.md:26]: Normally I would be fine with exposing them publicly and seeing if it becomes a problem, but some of the forms have dropdowns with private data in them. It is possible to restrict access to the forms with a password, but I think that'll probably ask for it every time, which will be annoying for the people who will be using this a lot. Let me experiment with that later today.  The other option is to replace the links to the forms with links to the tables, which  requires authentication and will keep you logged in throughout your session.

Comment (michaelkohn) [reply content/en/contribute/medic/product-development-process/ux-research-repo/_index.md:26]: Interestingly... I was able to password protect each form and it only every asked for the password once (for each form) as long as i didn't change the password, which is "convenient" for our use case. That said, rather than complicate it, i'd suggest removing the links and letting folks access in a way that is meaningful for them. 



Screenshot 2023-01-19 at 1 13 20 PM
way.

Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/ux-research-repo/_index.md]: Thanks both! I agree with removing the links, I don't think they bring much value to the documentation. The key concepts such as `organisation` and `nuggets` are explained in the `/key-concepts` page, and I feel that it is sufficient to give context to the readers. As a consequence, I removed the links.

**Review by michaelkohn — DISMISSED** [review]

**Comment by andrablaj** [comment]
@michaelkohn the Product Trio Activities page is still [out of date](https://github.com/medic/cht-docs/pull/918#pullrequestreview-1248244671). We can create a separate issue and review it whenever you or @mrjones-plip have some time.

**Comment by michaelkohn** [comment]
Oops. Did I just approve everything? I thought i was just approving the one tiny "remove baserow links" commit :-/

**Comment by andrablaj** [comment]
That's fine @michaelkohn, please take your time and suggest as many updates as needed, we are not in a hurry. And thank you for helping out.

File: .github/CODEOWNERS
Comment (garethbowen) [inline .github/CODEOWNERS:5]: I'm not a fan of CODEOWNERS files because it goes against the spirit of openness, restricts improvements, blocks improvements if people are on holiday, and tends to request reviews from many people when one would do.

We've tried them in the past and removed them because of these failings. Why do we think it'll work better this time?

File: content/en/contribute/medic/product-development-process/_index.md
Comment (garethbowen) [inline content/en/contribute/medic/product-development-process/_index.md:9]: For internal links throughout this PR it's better to use `{{% ref "..." $}}` rather than a regular link. This provides better checking for broken links at build time.

```suggestion
The Medic Product Team operates in a way that is most commonly referred to as [Continuous Discovery]({{% ref "continuous-discovery-overview" %}}). That process helps us to focus on the impact of our work and keep teammates well-aligned with the people who use the CHT.
```

File: content/en/contribute/medic/product-development-process/code-health.md
Comment (garethbowen) [inline content/en/contribute/medic/product-development-process/code-health.md:13]: There is no code health week anymore as far as I'm aware...

```suggestion
```

File: content/en/contribute/medic/product-development-process/testing/_index.md
Comment (garethbowen) [inline content/en/contribute/medic/product-development-process/testing/_index.md:7]: Unless you're still working on this I think it'd be better to have no page at all rather than an empty page.

**Review by garethbowen — CHANGES_REQUESTED** [review]
Great work on this! Thanks.

I've put a few minor suggestions inline. My main concern is about adding the CODEOWNERS file.

**Review by garethbowen — APPROVED** [review]
Nice one. I think it's good to go!

**Comment by andrablaj** [comment]
@abbyad @michaelkohn @mrjones-plip any additional comments on this PR, or can we merge this first version?

File: content/en/contribute/medic/product-development-process/ux-research-repo/adding-nuggets.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/ux-research-repo/adding-nuggets.md:20]: We haven't actually published "quick links" anywhere on the linked/ref page, and we don't really want to make them public since some of the data entry pages include drop downs with PII.  On the UX Research Repo page we say... 

> "To gain access to the UX Research Repo, please reach out in the #product-owner-chatter Slack channel. The access to the repository is currently restricted to the Medic team." 

Perhaps we can say something similar here?

File: content/en/contribute/medic/product-development-process/ux-research-repo/key-concepts.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/ux-research-repo/key-concepts.md:32]:
```suggestion

## Leads

Product suggestions or requests that did _not_ necessarily come directly from a product trio member interviewing or observing a user. These are often taken from forum posts or other 3rd party means.

**Example**: *Forum Post: It would be nice if the CHT used comic sans font* 

```

File: content/en/contribute/medic/product-development-process/ux-research-repo/key-concepts.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/ux-research-repo/key-concepts.md:34]:
```suggestion

## Leads

Product suggestions or requests that did _not_ necessarily come directly from a product trio member interviewing or observing a user. These are often taken from forum posts or other 3rd party means.

**Example**: *Forum Post: It would be nice if the CHT used comic sans font* 

```

File: content/en/contribute/medic/product-development-process/ux-research-repo/synthesizing-nuggets.md
Comment (michaelkohn) [inline content/en/contribute/medic/product-development-process/ux-research-repo/synthesizing-nuggets.md:11]:
```suggestion
2. Product trio and stakeholders review Nuggets quarterly
```

**Comment by michaelkohn** [comment]
I made a few quick suggestions but am otherwise fine to merge this version!

File: content/en/contribute/medic/product-development-process/ux-research-repo/adding-nuggets.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/ux-research-repo/adding-nuggets.md]: Great find, thanks @michaelkohn !

**Review by michaelkohn — APPROVED** [review]

**Comment by andrablaj** [comment]
Thanks @michaelkohn! 
@mrjones-plip I prefer to wait for your review before merging. Thank you! 🙏

File: content/en/contribute/medic/product-development-process/_index.md
Comment (mrjones-plip) [inline content/en/contribute/medic/product-development-process/_index.md:14]: Given the recent feedback on having more transparency into the product team, I think we should improve this line:
```suggestion
Each Focused Working Group has defined Objectives and Key Results (OKRs), which set their focus for a given quarter. These, in turn, are derived from the Product Team's yearly Objective Goals Strategies Measurements (OGSMs).  The [Schedule of Activities]({{% ref "contribute/medic/product-development-process/schedule-of-events" %}}) has more on the timing of all this through out any given year.
```

Then add OKR and OGSM to `contribute/medic/product-development-process/schedule-of-events`!

**Review by mrjones-plip — APPROVED** [review]
This is great, and really well thought out - thank you!

Gave some minor feedback about spelling out OKR and adding OGSMs.  Both of which I think are pertinent as we try and get more folks to understand why we do stuff when we do it :rocket: 

as it's minor, approving now so you can update as you like and go live

File: content/en/contribute/medic/product-development-process/_index.md
Comment (andrablaj) [reply content/en/contribute/medic/product-development-process/_index.md]: Great one! Thank you for the addition

**Pull request merged**
